src/main/kotlin/app/revanced/patches/youtube/layout/hide/loadmorebutton/bytecode/patch/HideLoadMoreButtonPatch.kt

```diff
@@ -0,0 +1,48 @@
+package app.revanced.patches.youtube.layout.hide.loadmorebutton.bytecode.patch
+
+import app.revanced.extensions.toErrorResult
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstruction
+import app.revanced.patcher.extensions.instruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.youtube.layout.hide.loadmorebutton.bytecode.fingerprints.HideLoadMoreButtonFingerprint
+import app.revanced.patches.youtube.layout.hide.loadmorebutton.resource.patch.HideLoadMoreButtonResourcePatch
+import org.jf.dexlib2.iface.instruction.formats.Instruction11x
+
+@Patch
+@Name("hide-load-more-button")
+@Description("Hides the button under videos to load similar videos.")
+@DependsOn([HideLoadMoreButtonResourcePatch::class])
+@Version("0.0.1")
+class HideLoadMoreButtonPatch : BytecodePatch(
+    listOf(
+        HideLoadMoreButtonFingerprint
+    )
+) {
+    private companion object {
+        const val INTEGRATIONS_CLASS_DESCRIPTOR = "Lapp/revanced/integrations/patches/HideLoadMoreButtonPatch;"
+    }
+
+    override fun execute(context: BytecodeContext): PatchResult {
+        HideLoadMoreButtonFingerprint.result?.let {
+            it.mutableMethod.apply {
+                val insertIndex = it.scanResult.patternScanResult!!.endIndex + 1
```

Comment:
What is the reason for choosing the opcode pattern, so that the insert index is out of range of the pattern?

Reply:
No reason, on next index a new view is initialized, so I thought that's it for it.

Reply:
In that case, decrease the size of the fingerprint. The opcode pattern should be reduced to a minimum as long as the other properties of the fingerprint match uniquely to the same method. The pattern should then be used to find the index relative to the matched index.